lib/hunter/status.ex

```diff
@@ -138,12 +140,26 @@ defmodule Hunter.Status do
   * `id` - status identifier
 
   """
-  @spec unreblog(Hunter.Client.t, non_neg_integer) :: Hunter.Status.t
+  @spec unreblog(Hunter.Client.t, status_id) :: Hunter.Status.t
   def unreblog(conn, id) do
     @hunter_api.unreblog(conn, id)
   end
 
   @doc """
+  Fetch the list of users who reblogged the status.
+
+  ## Parameters
+
+    * `conn` - connection credentials
+    * `id` - status identifier
+
+  """
+  @spec reblogged_by(Hunter.Client.t, non_neg_integer) :: [Hunter.Account.t]
```

Comment:
Alright :)
I ping'd you the Elixir Slack, btw. So you can reach me if you need.